packages/oidf-client/src/agent/OIDFClient.ts

```diff
@@ -1,81 +1,70 @@
-import { IAgentPlugin } from '@veramo/core'
-import { CreateJwsCompactArgs, IJwsValidationResult, JwtCompactResult, VerifyJwsArgs } from '@sphereon/ssi-sdk-ext.jwt-service'
-import { IOIDFClient, OIDFClientArgs, RequiredContext, ResolveTrustChainArgs, ResolveTrustChainCallbackResult } from '../types/IOIDFClient'
-import { com } from '@sphereon/openid-federation-client'
-import { schema } from '../index'
-import { JWK } from '@sphereon/ssi-types'
-import FederationClient = com.sphereon.oid.fed.client.FederationClient
-import DefaultFetchJSImpl = com.sphereon.oid.fed.client.fetch.DefaultFetchJSImpl
-import DefaultTrustChainJSImpl = com.sphereon.oid.fed.client.trustchain.DefaultTrustChainJSImpl
-import DefaultCallbacks = com.sphereon.oid.fed.client.service.DefaultCallbacks
+import {FederationClient, ICryptoService, IFetchService} from '@sphereon/openid-federation-client'
+import {JWK} from "@sphereon/ssi-types";
+import {IAgentPlugin} from '@veramo/core'
+import {Request} from "cross-fetch";
```

Comment:
Can it still find this? "cross-fetch": "^3.1.8" is not in devDependencies anymore. 
(Maybe do "pnpm prune" followed by "pnpm build:clean" then run the test to confirm.